src/components/pokemon-details/pokemon-details.component.jsx

```diff
@@ -11,24 +11,28 @@ const PokemonDetails = ({ id, types, name, weight, height }) => {
 
   // TODO: not super-happy with this approach. Think about refactor
   // Because the api has the types in reverse order
+
   for (let i in types) {
-    if (types[i].slot === 1) {
-      typeMain = types[i].type.name;
-    } else if (types[i].slot === 2) {
-      typeSecondary = types[i].type.name;
+    switch(types[i].slot) {
+      case 2:
+        typeSecondary = types[i].type.name;
+        break;
+      case 1:
+      default:
+        typeMain = types[i].type.name;
```

Comment:
nit: something a bit different. not really an improvement. with only 2 cases both versions have minimal speed difference.

Reply:
I tried something a little different. I did a if type was 2 then assign to typeSecondary, else assign to 1. A tad shorter.